backend/src/helpers/index.ts

```diff
@@ -0,0 +1,17 @@
+import { paramCase, Options } from 'change-case';
+import { Case } from '~/common/enums/case/case.enum';
+
+const caseTypeToFn: Record<Case, typeof paramCase> = {
+  'kebab': paramCase,
+};
+const changeStringCase = (
+  string: string,
+  caseType: Case,
+  options?: Options,
+  // eslint-disable-next-line max-params
+): string => {
```

Comment:
If there is more than 2 parameters for fn - it's better to provide arguments as object
```suggestion
type Args = {
  stringToChange: string,
  caseType: Case,
  options?: Options,
};
const changeStringCase = ({
  stringToChange: string,
  caseType: Case,
  options?: Options,
}: Args): string => {
```